src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  InternshipProject,
+  LanguageEngagement,
+  TranslationProject,
+} from '../../../api/schema.generated';
+import { Breadcrumb } from '../../../components/Breadcrumb';
+import { BudgetOverviewCard } from '../../../components/BudgetOverviewCard';
+import { CardGroup } from '../../../components/CardGroup';
+import { LanguageEngagementListItemCard } from '../../../components/LanguageEngagementListItemCard';
+import { PartnershipSummary } from '../../../components/PartnershipSummary';
+import { ProjectMembersSummary } from '../../../components/ProjectMembersSummary';
+import { useProjectOverviewQuery } from './ProjectOverview.generated';
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+    '& > *': {
+      marginBottom: spacing(3),
+    },
+  },
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  uploadInput: {
+    display: 'none',
+  },
+  uploadButton: {
+    backgroundColor: palette.primary.main,
+    color: palette.common.white,
+    marginRight: spacing(1),
+    '&:hover': {
+      backgroundColor: fade(palette.primary.main, 0.5),
+    },
+  },
+  chips: {
+    '& > *': {
+      marginRight: spacing(2),
+    },
+  },
+  budgetOverviewCard: {
+    marginRight: spacing(3),
+  },
+  addButton: {
+    marginLeft: 'auto',
+    marginRight: spacing(1),
+    backgroundColor: palette.error.main,
+    color: palette.common.white,
+    '&:hover': {
+      backgroundColor: fade(palette.error.main, 0.5),
+    },
+  },
+}));
+
+export const ProjectOverview: FC = () => {
+  const classes = useStyles();
+  const { data, error } = useProjectOverviewQuery();
+  const { pathname } = useLocation();
+  const project = data?.project as
+    | TranslationProject
+    | InternshipProject
+    | undefined;
+
+  const languageEngagementSection = (project?: TranslationProject) => {
+    return (
+      <>
+        <div className={classes.container}>
+          <Typography variant="h3">
+            {project?.engagements?.total} Language Engagements
+          </Typography>
+          <IconButton
+            classes={{ root: classes.addButton }}
+            aria-label="add button"
+          >
+            <Add />
+          </IconButton>
+          <Typography variant="h4">Add Language</Typography>
+        </div>
+        {project?.engagements?.items?.map((engagement) => {
+          const languageEngagement = engagement as LanguageEngagement;
+          return languageEngagement?.language ? (
+            <LanguageEngagementListItemCard {...languageEngagement} />
+          ) : null;
+        })}
+      </>
+    );
+  };
+
+  return (
+    <div className={classes.root}>
+      {error ? (
+        <Typography variant="h4">Error fetching Project</Typography>
+      ) : (
+        <>
+          <Breadcrumbs>
+            <Breadcrumb to="/projects">Project</Breadcrumb>
+            <Breadcrumb to={pathname}>{project?.name?.value}</Breadcrumb>
```

Comment:
It would be better to just define the path here instead of listening to route changes.
```suggestion
            <Breadcrumb to={`/projects/${id}`}>{project?.name?.value}</Breadcrumb>
```

Reply:
Path now has id from params